### DISCUSSION TITLE
problem sending mails with mailer

### DISCUSSION THREAD

**Discussion by souna23** [symfony/symfony#48698] [Q&A]
You must configure a "Symfony\Component\Mime\BodyRendererInterface" when a "Symfony\Bridge\Twig\Mime\TemplatedEmail" instance has a text or HTML template set.

**Comment by MatTheCat** [comment]
Please don’t just copy-paste exceptions message here; you can follow [the issue form template](https://github.com/symfony/symfony/blob/52b5bb723b5b992cebfd13fd8286f9d83bd19d62/.github/ISSUE_TEMPLATE/1_Bug_report.yaml) instead.

Anyways I guess this has been fixed by #48587 on 6.2.2
